node/src/command.rs

```diff
@@ -55,14 +55,15 @@ impl SubstrateCli for Cli {
             "" | "dev" => Box::new(chain_spec::development_config()),
             "local_poa_testnet" => Box::new(chain_spec::local_testnet_config()),
             "poa_testnet" => Box::new(chain_spec::testnet_config()),
+            "dock_mainnet" => Box::new(chain_spec::mainnet_config()),
```

Comment:
Edit: I guess it's kind-of a mute point because users will need to specify a path to a chainspec anyway, so the never acually have to type out `dock-node --chain dock_mainnet`. Instead it would be more like
`dock-node --chain ./cspec/mainnet.json`